src/fontra/views/editor/editor.js

```diff
@@ -312,14 +312,16 @@ export class EditorController {
               title: "Zoom to fit",
               enabled: () => {
                 let viewBox = this.sceneController.getSelectionBox();
-                const size = rectSize(viewBox);
-                if (size.width < 4 && size.height < 4) {
-                  const center = rectCenter(viewBox);
-                  viewBox = centeredRect(center.x, center.y, 10, 10);
-                } else {
-                  viewBox = rectAddMargin(viewBox, 0.1);
+                if (viewBox) {
```

Comment:
Since "enabled" is supposed to return a bool, I would reverse this:
```js
    if (!viewBox) {
      return false;
    }
```
(That also makes the diff to the original smaller)

Reply:
Ahh, I really should have done that. 😅